### PR TITLE
feat(footer): introduce cookie preference placeholder

### PR DESCRIPTION
**Pull request by asudoh**
### Related Ticket(s)

Refs #3583.

### Description

This change adds `<dds-legal-nav-cookie-perferences-placeholder>`, a placeholder for cookie preference link.

### Changelog

**New**

- `<dds-legal-nav-cookie-perferences-placeholder>`, a placeholder for cookie preference link.

### CODE REVIEW

File: packages/web-components/src/components/footer/legal-nav-cookie-preferences-placeholder.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  render() {
+    const { _handleSlotchange: handleSlotchange } = this;
+    return html`
+      <slot @slotchange="${handleSlotchange}"></slot>
+    `;
+  }
+
+  static get stableSelector() {
+    return `${ddsPrefix}--privacy-cp`;
```
Comment (jeffchew) [inline packages/web-components/src/components/footer/legal-nav-cookie-preferences-placeholder.ts:60]: @asudoh this doesn't seem to be working. I checked and looks like the stable selector is being set as `data-auto-id` and not `data-autoid` (no extra dash between `auto` and `id`)

Comment (asudoh) [reply packages/web-components/src/components/footer/legal-nav-cookie-preferences-placeholder.ts:60]: Oh I see; Should such change be applied globally, or just here?

Comment (asudoh) [reply packages/web-components/src/components/footer/legal-nav-cookie-preferences-placeholder.ts:60]: NVM - I saw `data-autoid` is used globally in React. Fixed: https://github.com/carbon-design-system/ibm-dotcom-library/pull/3624/commits/6e455b754b74cfd872af19551e1a1171b93e8d44

**Comment by jeffchew** [comment]
I tried and the link doesn't seem to be getting loaded in. Going to see what's going on there.

**Comment by asudoh** [comment]
Superseded by: #3656

**Pull request closed without merging**